src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -142,6 +142,16 @@ public VerboseProcess(final Process prc, final Level stdout,
         if (stderr == null) {
             throw new IllegalArgumentException("stderr LEVEL can't be NULL");
         }
+        if (Level.ALL.equals(stdout)) {
+            throw new IllegalArgumentException(
+                "stdout LEVEL can't be set to ALL"
```

Comment:
@mkordas do you think such Javadoc will be ok?:

     * Public ctor, with a given process and logging levels for {@code stdout}
     * and {@code stderr}. Neither {@code stdout} nor {@code stderr} cannot be
     * set to {@link Level#ALL} because it is intended to be used only for
     * internal configuration.

Do you think we should change the error message as well? For example to

     stdout LEVEL can't be set to ALL because it is intended only for internal configuration

?